lldb/tools/lldb-dap/package.json

```diff
@@ -88,6 +88,12 @@
           "additionalProperties": {
             "type": "string"
           }
+        },
+        "lldb-dap.serverMode": {
+          "scope": "resource",
+          "type": "boolean",
+          "description": "Run lldb-dap in server mode. When enabled, lldb-dap will start a background server that will be reused between debug sessions. This can improve launch performance and caching of debug symbols between debug sessions.",
```

Comment:
I think we can make this a stronger statement. Maybe something like: "This allows caching of things like debug symbols and should improve launch performance".

Reply:
Updated the wording, let me know what you think.